docs/docs.markdown

```diff
@@ -41,37 +41,37 @@ Now you should have reachable Strapi instance here [http://localhost:1337/](http
 
 Go to Content-Type Build in your admin instance [http://localhost:1337/admin/plugins/content-type-builder/](http://localhost:1337/admin/plugins/content-type-builder/) and then create Collection as below:
 
-![image](/assets/images/s-01.jpg)
+![image](../assets/images/s-01.jpg)
```

Comment:
What is the purpose of this?

Reply:
If I remember, the images do not load on launch of localhost site. 

I think i was because of the jekyll-strapi baseurl.